jac-uk/admin
# #2082 Agency ACRO report - additional fields

## What's included?

- Add the following fields to the agency ACRO report
    - Middle Name
    - Previous known name(s)
    - Postcode
- Fix lint error

Closes #2082 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Example exercise: https://jac-admin-develop--pr2083-feature-2082-agency-xrx44k5p.web.app/exercise/ZutEH7OHcjRlUIfXLyqP/reports/agency

1. Go to applications and ensure the above fields are not empty.
1. Go to the Agency Report of an exercise.
2. Click `Refresh` and `Export data` buttons to export the report.
3. Check if the above fields appear in the report. 

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, video demo, notes etc.

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required
- Permissions have been added / updated. Details:

---
PREVIEW:PRODUCTION
_can be OFF, DEVELOP or STAGING_
